src/v/pandaproxy/schema_registry/test/test_json_schema.cc

```diff
@@ -261,6 +261,12 @@ static constexpr auto compatibility_test_cases = std::to_array<
     .writer_schema = R"({"type": "array", "maxItems": 11})",
     .reader_is_compatible_with_writer = false,
   },
+  // array checks: uniqueItems must be compatible
+  {
+    .reader_schema = R"({"type": "array", "uniqueItems": true})",
+    .writer_schema = R"({"type": "array"})",
+    .reader_is_compatible_with_writer = false,
+  },
```

Comment:
nit: is it worth adding a test here for the default value handling?

Reply:
yes, i'll followup